### DISCUSSION TITLE
Starknet-Foundry Developers call

### DISCUSSION THREAD

**Discussion by Arcticae** [foundry-rs/starknet-foundry#978] [Announcements]
## This is an event for a recurring bi-weekly meeting.

Developers who want to ask questions about implementation, work on features, suggest improvements are welcome to join.

## Next Meeting Schedule:
Friday, October 27, 12PM UTC

Link: [zoom](https://us05web.zoom.us/j/89555589815?pwd=XkLsWO4RjIYE1YaO2MNdVZd0traRji.1)

**Comment by cptartur** [comment]
New link for the meeting: https://us04web.zoom.us/j/77400481122?pwd=iAvoxg53minWl5q4U9jJxpkkc8VBJE.1

**Comment by cptartur** [comment]
## Updates (27.10.2023)

### In Upcoming Release

- Deployment scripts in Cast (not all functionalities)
- `sncast verify` command

### In Progress

- Gas estimation
- Global cheatcodes
- Parallel execution code simplification
- Basic benchmarking of forge codebase
- Refactors, refactors, refactors

### In Research

- Mapping Cairo<>Sierra code to enable work on more advanced features like profiling, test coverage etc.
